lib/libutils/isoc/include/sys/_default_fcntl.h

```diff
@@ -0,0 +1,72 @@
+#ifndef	_SYS__DEFAULT_FCNTL_H_
```

Comment:
this header and the other newlib imported file need a copyright/licence notice + an SPDX-License-Identifer tag. It's seems to me this is under BSD 2 or 3 clause license, copyrighted by RedHat and others: https://github.com/eblot/newlib/blob/master/COPYING.NEWLIB.

Reply:
Add SPDX-License-Identifer tag in https://github.com/OP-TEE/optee_os/pull/6700/commits/c4e2b5c57f827f7ad3c66e9c05fd57c3d555a250